t/lib/MyModel.pm

```diff
@@ -30,7 +30,6 @@ sub testing {
     if ( $model->es_version < 1 ) {
         plan skip_all => 'Requires Elasticsearch 1.0.0';
```

Comment:
Are these changes compatible with 1.x or should we just update this to require 2.x?  I'm fine with not jumping through hoops to support anything less than 2.x